oat-sa/extension-tao-testqti
# Feature/nex 360 e2e test for order and inline interactions

Add inline interaction and order interaction tests.

Note:
` views/js/e2e/_helpers/selectors/interactionSelectors.js` is copied from https://github.com/oat-sa/extension-tao-testqti/blob/feature/nex-183/e2e-test-for-interactions/views/js/e2e/_helpers/selectors/interactionSelectors.js, to avoid later conflicts.

Test:
- Have a working `package-tao`.
- Go to `tao/views/build/node_modules/@oat-sa/tao-e2e-runner`
- Create `cypress.env.json` from `cypress.env.sample.json` and it should contain `testFiles` like this: `"testFiles": "taoQtiTest/views/js/e2e/**/interactions/**/*.spec.js",`
- `npm run test`
- all tests should pass
- all example item, test and delivery should be deleted
